.java-version

```diff
@@ -1 +1 @@
-1.8.0.232
+1.8.0.242
```

Comment:
Was there a particular issue that prompted this?

Reply:
No, it's a mistake.  Reverted.